.github/workflows/package.yml

```diff
@@ -247,6 +247,25 @@ jobs:
           GH_TOKEN: ${{ github.token }}
 
       # Test
+      - name: Install Incus
+        if: |
+          startsWith(env.OS, 'almalinux-')
```

Comment:
Use Incus on only AlamaLinux at first because we want to test it on this OS.

Reply:
Since it doesn't seem hard to migrate to testing on Incus for all OS, could you target all OS?

Reply:
Sure.

Reply:
fix: [c3fd1e0](https://github.com/pgroonga/pgroonga/pull/511/commits/c3fd1e04436e357c03431c13c588a1867d90fb5a)